tox.ini

```diff
@@ -6,6 +6,7 @@ deps =
     coverage
     pretend
     pytest
+    py27,pypy: pyblake2
```

Comment:
`hashlib` on py2 doesn't have a `blake2b` hash but `pyblake2` is used for backwards compatibility there so let's install it for our testing purposes.